packages/react-component-library/src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -925,6 +932,61 @@ describe('DatePicker', () => {
         'yyyy/mm/dd'
       )
     })
+
+    describe('when a valid date is typed', () => {
+      beforeEach(async () => {
+        const input = wrapper.getByTestId('datepicker-input')
+
+        userEvent.type(input, `2016/02/03{enter}`)
+      })
+
+      it('invokes the `onChange` callback', () => {
+        const expectedDate = new Date('2016-02-03T12:00:00.000Z')
+        expect(onChange).toHaveBeenCalledTimes(1)
+        expect(onChange).toHaveBeenCalledWith({
+          startDate: expectedDate,
+          endDate: expectedDate,
+        })
+      })
+
+      it("isn't in an error state", () => {
```

Comment:
Single quotes here

Reply:
Nested single quote there – Prettier will change it even if you escape it.